packages/lib/src/vis/shared/RatioEnforcer.tsx

```diff
@@ -2,14 +2,10 @@ import { useThree } from '@react-three/fiber';
 import { useEffect } from 'react';
 
 import { useMoveCameraTo } from '../../interactions/hooks';
+import { useVisCanvasContext } from './VisCanvasProvider';
 
-interface Props {
-  visRatio: number | undefined;
-}
-
-function RatioEnforcer(props: Props) {
-  const { visRatio } = props;
-
+function RatioEnforcer() {
+  const { visRatio } = useVisCanvasContext();
```

Comment:
Getting `visRatio` from context to remove the prop (we do this in `AxisSystem` too.